src/Symfony/Component/HttpClient/Internal/CurlClientState.php

```diff
@@ -53,7 +53,7 @@ public function __construct(int $maxHostConnections, int $maxPendingPushes)
             curl_multi_setopt($this->handle, \CURLMOPT_PIPELINING, \CURLPIPE_MULTIPLEX);
         }
         if (\defined('CURLMOPT_MAX_HOST_CONNECTIONS')) {
```

Comment:
let's keep the defaults when maxHostConnections is not set?
```suggestion
        if (\defined('CURLMOPT_MAX_HOST_CONNECTIONS') && 0 < $maxHostConnections) {
            $maxHostConnections = curl_multi_setopt($this->handle, \CURLMOPT_MAX_HOST_CONNECTIONS, $maxHostConnections)) ? 2147483647 : $maxHostConnections;
```

Reply:
Done https://github.com/symfony/symfony/compare/8fe2f2fe3f234a742e784befa5318da5a4c52e46..4f9b5decd96a9d0af2749ad83e6ce1f4a6608177